config.py

```diff
@@ -325,6 +325,17 @@ def get_histogram_percentiles(configstr=None):
     return result
 
 
+def get_histogram_submit_methods(use_histogram=False, tags_to_strip=None):
+    from checks import AgentCheck
```

Comment:
`generate_histogram_func` and `AgentCheck.generate_historate_func` are going to be remove in agent6. We should stop using them. They don't need to be linked with the AgentCheck class.

see: https://github.com/DataDog/integrations-core/pull/750/files